bin/xud

```diff
@@ -154,6 +154,11 @@ const { argv } = require('yargs')
       default: undefined,
       alias: 't',
     },
+    'p2p.torSocks': {
```

Comment:
What about `socksport`? It could also be `torsocksport` or `torport`. Either way I think the "port" qualifier is helpful.

Even if you'd rather keep the name, we should make it all lower caps (`torsocks`) just to be consistent with the other arg names.

Reply:
I ended up going with `torport`.